### DISCUSSION TITLE
Your browser, your way

### DISCUSSION THREAD

**Discussion by GunGunGun** [zen-browser/desktop#1952] [General]
@wolfbeast It looks like Samsung violated copyright trademark of "Your browser, your way", can you take a look ?

![image](https://github.com/user-attachments/assets/af69051d-be24-42e2-a786-d07b8f265ef5)

Zen developer've updated their website to respect copyright trademark:
![image](https://github.com/user-attachments/assets/1af152be-2ba2-4eb5-afd3-be0f60d18993)

**Comment by devxan** [comment]
While I still doubt the authenticity of that tagline's trademarked or copyrighted status, this was changed for Zen Browser in #1931. 

I think this is a bad place to attempt contact at Pale Moon developers/team, especially for an entirely different application and company.

**Comment by GunGunGun** [comment]
[Zen Browser uses alleged Pale Moon-owned trademark "Your Browser, Your Way"](https://old.reddit.com/r/palemoon/comments/1fxo5gq/zen_browser_uses_alleged_pale_moonowned_trademark/)
[Just got into a copyright issue, any advise?](https://old.reddit.com/r/opensource/comments/1fxnrwx/just_got_into_a_copyright_issue_any_advise/)
